multipass/launch_test.go

```diff
@@ -0,0 +1,18 @@
+package multipass
+
+import (
+	"testing"
+)
+
+func TestLaunch(t *testing.T) {
+
+	_, err := LaunchV2(&LaunchReqV2{
+		CPUS: "2",
+		Memory: "2GiB",
+		Name: "instanceName", 
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+
```

Comment:
We need to stop the instance too.

Reply:
ok, I will stop the instance and I’ll double check about the -m option